diff --git a/node_modules/moti/build/core/index.js b/node_modules/moti/build/core/index.js
index 6c5a5fa..fca3f0e 100644
--- a/node_modules/moti/build/core/index.js
+++ b/node_modules/moti/build/core/index.js
@@ -1,5 +1,5 @@
 export { default as motify } from './motify';
-export { AnimatePresence } from 'framer-motion';
+export { AnimatePresence } from 'framer-motion/dist/framer-motion';
 export * from './types';
 export { default as useAnimationState } from './use-animator';
 export { default as useDynamicAnimation } from './use-dynamic-animation';
diff --git a/node_modules/moti/build/core/motify.js b/node_modules/moti/build/core/motify.js
index a3a0956..c133f44 100644
--- a/node_modules/moti/build/core/motify.js
+++ b/node_modules/moti/build/core/motify.js
@@ -1,5 +1,5 @@
 import { jsx as _jsx } from "react/jsx-runtime";
-import { usePresence, PresenceContext } from 'framer-motion';
+import { usePresence, PresenceContext } from 'framer-motion/dist/framer-motion';
 import { forwardRef, useContext, } from 'react';
 import Animated from 'react-native-reanimated';
 import { useMotify } from './use-motify';
