diff --git a/node_modules/nativewind/dist/styled/use-interaction.js b/node_modules/nativewind/dist/styled/use-interaction.js
index a66e32b..c36e7a8 100644
--- a/node_modules/nativewind/dist/styled/use-interaction.js
+++ b/node_modules/nativewind/dist/styled/use-interaction.js
@@ -11,21 +11,7 @@ function useInteraction(dispatch, mask, props) {
         const isParentOrGroup = (0, selector_1.matchesMask)(mask, selector_1.PARENT) || (0, selector_1.matchesMask)(mask, selector_1.GROUP);
         const handlers = {};
         if (isParentOrGroup || (0, selector_1.matchesMask)(mask, selector_1.ACTIVE)) {
-            if (react_native_1.Platform.OS === "web") {
-                handlers.onMouseDown = function (event) {
-                    if (ref.current.onMouseDown) {
-                        ref.current.onMouseDown(event);
-                    }
-                    dispatch({ type: "active", value: true });
-                };
-                handlers.onMouseUp = function (event) {
-                    if (ref.current.onMouseUp) {
-                        ref.current.onMouseUp(event);
-                    }
-                    dispatch({ type: "active", value: false });
-                };
-            }
-            else {
+            if (react_native_1.Platform.OS === "ios" || react_native_1.Platform.OS === "android") {
                 handlers.onPressIn = function (event) {
                     if (ref.current.onPressIn) {
                         ref.current.onPressIn(event);
@@ -38,20 +24,33 @@ function useInteraction(dispatch, mask, props) {
                     }
                     dispatch({ type: "active", value: false });
                 };
+            } else {
+                handlers.onPointerDown = function (event) {
+                    if (ref.current.onPointerDown) {
+                        ref.current.onPointerDown(event);
+                    }
+                    dispatch({ type: "active", value: true });
+                };
+                handlers.onPointerUp = function (event) {
+                    if (ref.current.onPointerUp) {
+                        ref.current.onPointerUp(event);
+                    }
+                    dispatch({ type: "active", value: false });
+                };
             }
         }
         if (isParentOrGroup || (0, selector_1.matchesMask)(mask, selector_1.HOVER)) {
-            handlers.onHoverIn = function (event) {
-                if (ref.current.onHoverIn) {
-                    ref.current.onHoverIn(event);
+            handlers.onPointerEnter = function (event) {
+                if (ref.current.onPointerEnter) {
+                    ref.current.onPointerEnter(event);
                 }
                 dispatch({ type: "hover", value: true });
             };
-            handlers.onHoverOut = function (event) {
-                if (ref.current.onHoverIn) {
-                    ref.current.onHoverIn(event);
+            handlers.onPointerLeave = function (event) {
+                if (ref.current.onPointerLeave) {
+                    ref.current.onPointerLeave(event);
                 }
-                dispatch({ type: "hover", value: true });
+                dispatch({ type: "hover", value: false });
             };
         }
         if (isParentOrGroup || (0, selector_1.matchesMask)(mask, selector_1.FOCUS)) {
